crates/nu-protocol/src/value/mod.rs

```diff
@@ -3879,6 +3933,134 @@ mod tests {
         }
     }
 
```

Comment:
Shouldn't be here `#[cfg(test)]`?

Reply:
The [enclosing module](https://github.com/nushell/nushell/blob/19fb9ad166003289cfc16f5551aa2f6a198498d8/crates/nu-protocol/src/value/mod.rs#L3859-L3860) is `#[cfg(test)]` already